datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -263,129 +274,135 @@ export default function DefaultPreviewCard({
 
     return (
         <PreviewContainer data-testid={dataTestID} onMouseDown={onPreventMouseDown}>
-            <LeftColumn expandWidth={!shouldShowRightColumn}>
-                <TitleContainer>
-                    <PlatformContentView
-                        platformName={platform}
-                        platformLogoUrl={logoUrl}
-                        platformNames={platforms}
-                        platformLogoUrls={logoUrls}
-                        entityLogoComponent={logoComponent}
-                        instanceId={platformInstanceId}
-                        typeIcon={typeIcon}
-                        entityType={type}
-                        parentContainers={parentContainers?.containers}
-                        parentNodes={parentNodes?.nodes}
-                        parentContainersRef={contentRef}
-                        areContainersTruncated={isContentTruncated}
-                    />
-                    <EntityTitleContainer>
-                        <Link to={url}>
-                            {previewType === PreviewType.HOVER_CARD ? (
-                                <CardEntityTitle onClick={onClick} $titleSizePx={titleSizePx}>
-                                    {name || ' '}
-                                </CardEntityTitle>
-                            ) : (
-                                <EntityTitle onClick={onClick} $titleSizePx={titleSizePx}>
-                                    {name || ' '}
-                                </EntityTitle>
+            <FirstRow>
```

Comment:
The changes to the jsx are:
* Wraps the left/right columns in a FirstRow
* Wraps the stats (subheader) in a SecondRow
* Moves the subHeader to the SecondRow

Reply:
this makes sense! I'd ask that before merging we do one more manual test to ensure things look good in all the different variations where there are stats, when there's no stats, when there's tags, terms, owners, lots of them, none of them, etc.

The reason I ask is because this component is notoriously fickle and lots of things rely on other things as you can see by the messy code below here...